Source/bmalloc/libpas/src/libpas/pas_report_crash.c

```diff
@@ -67,6 +67,15 @@ static PAS_ALWAYS_INLINE kern_return_t PAS_WARN_UNUSED_RETURN pas_update_report_
     return KERN_SUCCESS;
 }
 
+/*
+ * Function to be called to check if PGM has been enabled on this process at any point,
+ * regardless of if it has since been disabled, or if any guarded allocations were made.
+*/
+kern_return_t pas_pgm_enabled_on_process(void)
```

Comment:
Please rename it to `pas_probabilistic_guard_malloc_enabled_on_process`.
And let's move this implementation to `pas_probabilistic_guard_malloc_allocator.c`.